tsdb/head.go

```diff
@@ -2345,3 +2345,15 @@ type noopSeriesLifecycleCallback struct{}
 func (noopSeriesLifecycleCallback) PreCreation(labels.Labels) error { return nil }
 func (noopSeriesLifecycleCallback) PostCreation(labels.Labels)      {}
 func (noopSeriesLifecycleCallback) PostDeletion(...labels.Labels)   {}
+
+func (h *Head) Size() int64 {
+	if h.wal != nil {
+		walSize, _ := h.wal.Size()
+		return walSize + h.chunkDiskMapper.Size()
+	}
+	return 0
```

Comment:
Because `h.chunkDiskMapper` would always be present
```suggestion
	walSize := 0
	if h.wal != nil {
		walSize, _ = h.wal.Size()	
	}
	return walSize + h.chunkDiskMapper.Size()
```

Reply:
thanks for catching that